api/tests/routes/backoffice/individual_bookings_test.py

```diff
@@ -979,57 +1006,124 @@ def test_batch_mark_as_used_cancelled_bookings(self, legit_user, authenticated_c
+        assert response.status_code == 303
+
+        redirected_response = authenticated_client.get(response.headers["location"])
+
+        alerts = ";".join(html_parser.extract_alerts(redirected_response.data))
+        assert "1 réservation a été validée" in alerts
+        assert "Certaines réservations n'ont pas pu être validées et ont été ignorées :" in alerts
+        assert f"- 1 réservation dont le crédit associé est insuffisant ({insufficient_funds_booking.token})" in alerts
+
+    def test_batch_mark_as_used_bookings_insufficient_stock(self, legit_user, authenticated_client):
+        insufficient_stock_booking = bookings_factories.BookingFactory(
+            status=bookings_models.BookingStatus.CANCELLED,
+            deposit=users_factories.DepositGrantFactory(),
+            stock__quantity=0,
         )
+        other_booking = bookings_factories.BookingFactory()
+        parameter_ids = str(insufficient_stock_booking.id) + "," + str(other_booking.id)
+
+        response = self.post_to_endpoint(authenticated_client, form={"object_ids": parameter_ids})
+        assert response.status_code == 303
+
+        redirected_response = authenticated_client.get(response.headers["location"])
+
+        alerts = ";".join(html_parser.extract_alerts(redirected_response.data))
+        assert "1 réservation a été validée" in alerts
+        assert "Certaines réservations n'ont pas pu être validées et ont été ignorées :" in alerts
         assert (
-            "Le solde d'au moins un des comptes jeune est insuffisant pour valider ces réservations"
-            in html_parser.extract_alert(response.data)
+            f"- 1 réservation dont l'offre n'a plus assez de stock disponible ({insufficient_stock_booking.token})"
+            in alerts
         )
 
-    def test_batch_mark_as_used_bookings_insuffisant_stock(self, legit_user, authenticated_client):
-        booking1 = bookings_factories.BookingFactory(
+    def test_batch_mark_as_used_bookings_with_multiple_errors(self, legit_user, authenticated_client):
+        cancelled_booking = bookings_factories.CancelledBookingFactory()
+        insufficient_stock_booking = bookings_factories.BookingFactory(
             status=bookings_models.BookingStatus.CANCELLED,
             deposit=users_factories.DepositGrantFactory(),
             stock__quantity=0,
         )
-        booking2 = bookings_factories.BookingFactory()
-        parameter_ids = str(booking1.id) + "," + str(booking2.id)
+        already_reimbursed_booking = bookings_factories.ReimbursedBookingFactory()
+        another_reimbursed_booking = bookings_factories.ReimbursedBookingFactory()
+        parameter_ids = (
+            str(cancelled_booking.id)
+            + ","
+            + str(insufficient_stock_booking.id)
+            + ","
+            + str(already_reimbursed_booking.id)
+            + ","
+            + str(another_reimbursed_booking.id)
+        )
```

Comment:
Proposition peut-être plus légère : 
```suggestion
        parameter_ids = f"{cancelled_booking.id},{insufficient_stock_booking.id},{already_reimbursed_booking.id},{another_reimbursed_booking.id}"
```

La même "optimisation" peut s'appliquer à `test_batch_cancel_booking_with_multiple_errors`.